### DISCUSSION TITLE
Wrong path to assets to load static files

### DISCUSSION THREAD

**Discussion by Standupic** [vercel/next.js#62086] [Help]
### Summary

I am trying to set up project on next.js nothing hard there use create-next-app. When i run project 

> npm run dev

The app hosts on localhost:3000 and the home page is loaded. But the css styles that are placed at .next/app/static/css/app/layout.css not found.

I see the next one:

![Снимок экрана 2024-02-15 в 13 19 06](https://github.com/vercel/next.js/assets/11669962/7ae12a73-0c06-4372-9722-265715f6d9f0)

![Снимок экрана 2024-02-15 в 13 21 11](https://github.com/vercel/next.js/assets/11669962/646cdc63-27c6-4af4-a54b-817486370b62)

I have been https://github.com/vercel/next.js/issues/8158 
Unfortunately I didn't find a solution to the problem.

Please help!

### Additional information

```js
"dependencies": {
    "@types/node": "20.11.17",
    "@types/react": "18.2.55",
    "@types/react-dom": "18.2.19",
    "autoprefixer": "10.4.17",
    "eslint": "8.56.0",
    "eslint-config-next": "13.4.5",
    "next": "13.4.5",
    "postcss": "8.4.35",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "tailwindcss": "3.4.1",
    "typescript": "5.3.3"
}
```

**Comment by Franetse7** [comment]
Ok
